session/index.js

```diff
@@ -143,6 +143,10 @@ class UserSession {
     this.storage.setSessionInfo(this.sessionInfo);
   }
 
+  getSessionId() {
+    return this.sessionInfo?.id || undefined; 
```

Comment:
This can potentially return the session ID from the last active session. @MoumitaM to help here.

Reply:
@saikumarrs I tried running this code locally, but I think the process has changed since. @MoumitaM is going to help me out here to test this and make sure it works (thank you so much). 
Side note: This is for Tommy john.

Reply:
Yes, time to update our developer docs in the repo CC @bardisg

Reply:
Updated the logic.